src/main/io/rc_controls.c

```diff
@@ -342,7 +343,16 @@ void updateActivatedModes(modeActivationCondition_t *modeActivationConditions)
         modeActivationCondition_t *modeActivationCondition = &modeActivationConditions[index];
 
         if (isRangeActive(modeActivationCondition->auxChannelIndex, &modeActivationCondition->range)) {
```

Comment:
This may be written as
```
if(rcChannelStability[AUX1 + modeActivationCondition->auxChannelIndex] == RX_CHANNEL_STABILITY_HIGH) {
   // update RC_MODE
} else {
  // keep old value of RC mode
}
```